.github/workflows/ci.yml

```diff
@@ -18,4 +18,6 @@ concurrency:
 jobs:
   puppet:
     name: Puppet
-    uses: voxpupuli/gha-puppet/.github/workflows/basic.yml@v1
+    uses: daberkow/gha-puppet/.github/workflows/beaker.yml@ubuntu24
```

Comment:
This will move to v3 on main repo once https://github.com/voxpupuli/gha-puppet/pull/57 is merged